src/basic/parse-util.h

```diff
@@ -11,6 +11,28 @@
 
 typedef unsigned long loadavg_t;
 
+#define TPM2_PCRS_MAX 24U
+
+typedef enum PcrIndex {
```

Comment:
could you add a comment here, something like this:

```
/* The following names for PCRs 0…7 are based on the names in the "TCG PC Client Specific Platform Firmware Profile Specification" (https://trustedcomputinggroup.org/resource/pc-client-specific-platform-firmware-profile-specification/) */

Reply:
> could you add a comment here, something like this:
> 
> ```
> /* The following names for PCRs 0…7 are based on the names in the "TCG PC Client Specific Platform Firmware Profile Specification" (https://trustedcomputinggroup.org/resource/pc-client-specific-platform-firmware-profile-specification/) */
> ```

You are awesome sir. Thank you

From what I initially opened as PR, to what this is now, after @bluca, first made it refined from its very crude form, to all the inputs you guys have now added, It's awesome. Thank you everyone.